pom.xml

```diff
@@ -152,6 +157,11 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpclient</artifactId>
+                <version>4.5.7</version>
+            </dependency>
```

Comment:
Does this plugin still use this dependency or not?
 https://github.com/jenkinsci/windows-azure-storage-plugin/blob/0e80f49fe57607d0ca41e2502aa66a150ac593d7/pom.xml#L180-L183

Reply:
This plugin does not depend on okhttp directly.